web/src/components/home/subcomponents/upcoming_events/event_cards/event_card.js

```diff
@@ -0,0 +1,68 @@
+import eventCardStyles from "./event_card.styles";
+import { Box, Divider, Typography, Button } from "@mui/material";
+import AccessTimeIcon from "@mui/icons-material/AccessTime";
+import FmdGoodOutlinedIcon from "@mui/icons-material/FmdGoodOutlined";
+
+const HeaderDetails = (props) => (
+  <Box>
+    {props.date && (
+      <Box sx={eventCardStyles.details}>
+        <AccessTimeIcon sx={eventCardStyles.icon} />
+        <Typography>{props.date.toString()}</Typography>
+      </Box>
+    )}
+    {props.location && (
+      <Box sx={eventCardStyles.details}>
+        <FmdGoodOutlinedIcon sx={eventCardStyles.icon} />
+        <Typography>{props.location}</Typography>
+      </Box>
+    )}
+  </Box>
+);
+
+const Footer = (props) => (
+  <Box>
+    <Divider sx={eventCardStyles.divider} />
+    <Box sx={eventCardStyles.footerBox}>
+      {props.price && (
+        <Box>
+          <Typography sx={eventCardStyles.priceLabel}>Price:</Typography>
+          <Typography sx={eventCardStyles.price}>{props.price}</Typography>
+        </Box>
+      )}
+      {props.link && (
+        <Button
+          sx={eventCardStyles.button}
+          variant="contained"
+          onClick={() => window.open(props.link)}
+        >
+          {"View Details"}
+        </Button>
+      )}
+    </Box>
+  </Box>
+);
+
+const EventCard = (props) => {
+  let link;
```

Comment:
Can we hardcode this to just be the FB link for now? We can update the code once we support other link types